VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTopic.java

```diff
@@ -106,4 +115,12 @@ private void appendTopicIds(List<String> ids) {
             child.appendTopicIds(ids);
         }
     }
+
+    public boolean isAllType() {
```

Comment:
Umm .. It isn't logically right e.g. discussionTopic.inAllTopics() translates to basic_question_types.inAllTopics() which is true literally but not logically :wink: